clevertap-core/src/main/java/com/clevertap/android/sdk/db/CtDatabase.kt

```diff
@@ -235,12 +236,12 @@ private val CREATE_EVENTS_TABLE = """
 private val CREATE_USER_EVENT_LOGS_TABLE = """
     CREATE TABLE ${Table.USER_EVENT_LOGS_TABLE.tableName} (
         ${Column.EVENT_NAME} STRING NOT NULL,
+        ${Column.NORMALIZED_EVENT_NAME} STRING NOT NULL,
```

Comment:
1. Do we need both event_name and normalised_event_name? Or can we normalise the event_name as soon the user raises it and use that throughout the code

2. Is it a product requirement for `getUserEventLog` to provide the original event_name as well? If we have both columns in the db, and the client calls
- recordEvent("test sample") 
- then recordEvent("Testsample")
- then getUserEventLog("test sample") 

Here the client might be expecting _countOfEvent as 1 and the eventName as "test sample"_ but will get _countOfEvent as 2 and eventName as Testsample_

Is this expected?

Reply:
Okayy.

1. recordEvent("test sample")
2. then recordEvent("Testsample")
3. then getUserEventLog("test sample")

Here the client might be expecting countOfEvent as 1 and the eventName as "test sample" but will get countOfEvent as 2 and eventName as Testsample. The returned data might be slightly inconsistent, if that is acceptable then can merge

Reply:
Recording events: `LeVeL PaSsEd`, `Level Passed`, and `levelpassed` all appear as `LeVeL PaSsEd` since this is the way the event was recorded for the first time (This is how currently backend works, if you push events and check user's profile activity on dashboard). So first variant of event is always considered as actual event and when you need counts it will be 3 as all events are same after normalization (removing white space and making smaller case).

Similarly pushing profile property <k:v> `MoViE: 1`, `moviE: 2`, and `movie: 3`  appear as MoViE: